src/main/kotlin/com/terraformation/backend/accelerator/ProjectAcceleratorDetailsService.kt

```diff
@@ -20,13 +20,18 @@ class ProjectAcceleratorDetailsService(
 
   /** Returns accelerator details for projects that are assigned to cohorts. */
   fun fetchParticipantProjectDetails(): List<ProjectAcceleratorDetailsModel> {
-    requirePermissions { readAllAcceleratorDetails() }
-
     return projectAcceleratorDetailsStore.fetch(COHORTS.ID.isNotNull()) {
       acceleratorProjectVariableValuesService.fetchValues(it)
     }
   }
 
+  /** Returns accelerator details for projects that are assigned to cohorts. */
+  fun fetchDetailsByCondition(condition: Condition): List<ProjectAcceleratorDetailsModel> {
```

Comment:
I think this creates a leaky abstraction. A _private_ "fetch by condition" function is fine as an implementation detail, but I don't think callers should concern themselves with the database schema.

For a simple situation like this, the difference isn't huge, but as a general piece of design philosophy in the code, I think blurring the architectural layers like this will make it harder to do things like introduce caching, add appropriate permission checks, write comprehensive tests, or stub out the fetch methods in tests.

Concretely, I'm suggesting there should be a `fetchDetailsByParticipantId` function and possibly a nullable version of `fetchOneById` since one of the later PRs in the stack is fetching details for a single project.